libusb/descriptor.c

```diff
@@ -1043,7 +1045,14 @@ int API_EXPORTED libusb_get_ssplus_usb_device_capability_descriptor(
 
 	// We can only parse the non-variable size part of the SuperSpeedPlus descriptor. The attributes
 	// have to be read "manually".
```

Comment:
This comment doesn't make much sense now that we are not "parsing" any longer.

Reply:
Please give me the comment text you'd like, and I'll update it.

Reply:
I was hoping you would confirm my interpretation of the comment :) I think by "parsing" it means using the old parse_descriptor function with a fixed conversion pattern. Now that the function is taken away, we are reading "manually" in any case. So I think the whole comment can be removed.